java/code/src/com/suse/manager/tools/DiskCheck.java

```diff
@@ -0,0 +1,72 @@
+/**
+ * Copyright (c) 2021 SUSE LLC
+ *
+ * This software is licensed to you under the GNU General Public License,
+ * version 2 (GPLv2). There is NO WARRANTY for this software, express or
+ * implied, including the implied warranties of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. You should have received a copy of GPLv2
+ * along with this software; if not, see
+ * http://www.gnu.org/licenses/old-licenses/gpl-2.0.txt.
+ *
+ * Red Hat trademarks are not licensed under GPLv2. No permission is
+ * granted to use or replicate Red Hat trademarks that are incorporated
+ * in this software or its documentation.
+ */
+
+package com.suse.manager.tools;
```

Comment:
as we discussed already, it might be a good idea to move this to the already existing "utils" package instead of creating a new one.
But I leave this up to you.

Reply:
I moved the class to the `utils` package and changed its name to `DiskCheckHelper` for consistency with the other members of the package.